tests/tool/Integration.Tests/E2ETest.cs

```diff
@@ -32,12 +32,12 @@ public E2ETest(ITestOutputHelper output)
             _output = output;
         }
 
-        [InlineData("AspNetSample/csharp", "TemplateMvc.csproj", "")]
-        [InlineData("WpfSample/csharp", "BeanTrader.sln", "BeanTraderClient.csproj")]
-        [InlineData("WpfSample/vb", "WpfApp1.sln", "")]
-        [InlineData("PCL", "SamplePCL.csproj", "")]
+        [InlineData("AspNetSample/csharp", "TemplateMvc.csproj", "", true)]
+        [InlineData("WpfSample/csharp", "BeanTrader.sln", "BeanTraderClient.csproj", true)]
+        [InlineData("WpfSample/vb", "WpfApp1.sln", "", false)]
```

Comment:
I don't see other files were changed regarding support for Visual Basic projects that use WPF. Does this return false through the unhandled exception flow?